src/main/scala/com/github/swagger/akka/SwaggerHttpService.scala

```diff
@@ -46,15 +46,15 @@ trait SwaggerGenerator {
   def basePath: String = "/"
   def apiDocsPath: String = "api-docs"
   def info: Info = Info()
-  def scheme: Scheme = Scheme.HTTP
+  def schemes: List[Scheme] = List(Scheme.HTTP)
```

Comment:
Yep, personally I'd suggest going for a new version, as having two ways of setting the scheme may be confusing to new users. 

I've updated my PR accordingly with the new version, however I'm happy to change to support both `scheme` and `schemes` if preferred.